src/main/java/org/elasticsearch/action/percolate/PercolateShardResponse.java

```diff
@@ -112,6 +118,10 @@ public InternalAggregations aggregations() {
         return aggregations;
     }
 
+    public List<SiblingReducer> reducers() {
+        return reducers;
+    }
```

Comment:
I think these reducers should be (de)serialized in readFrom/writeTo? Maybe it's worth having a percolation with reducers test to make sure everything works fine.